app/models/course/lesson_plan/item.rb

```diff
@@ -40,6 +40,17 @@ def copy_attributes(other, time_shift)
     self.end_at = other.end_at + time_shift if other.end_at
   end
 
+  # Test if the lesson plan item has started for self directed learning.
+  #
+  # @return [Boolean]
+  def self_directed_started?
```

Comment:
Would it be good practice to override `started?` instead, or should retain `self_directed_started?`